### PR TITLE
Added english user doc baby

### PR DESCRIPTION
**Pull request by RiceBowlJr**
In my **great great kindness**, here is my small present to _LinShare_ team, the wanted english version of my modest user manual. Feel free to comment and I will modify it (_**if**_ I accept the modifications indeed).
Bye buddies.
Alex.

### CODE REVIEW

File: documentation/EN/user/linshare-user.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+> Note :<br/>
+This user guide is only for 2.x LinShare versions.
+
+<a name="presentation">
+
+#### Overwall presentation
+
+</a>
+
+At the loin page, you need to choose your language, that won't be possible later though. The language list is available at the upper right corner. Once logged in, you arrive on your home page.
```
Comment (fmartin-linagora) [inline documentation/EN/user/linshare-user.md]: login

File: documentation/EN/user/linshare-user.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+- *UPLOAD & SHARE*: which redirect you to your [current uploads](#current-uploads).
+- *WORKGROUP*: which redirect you to your [shared space](#shared-space).
+
+<a name="my-space">
+
+##### My space
+
+</a>
+
+My space is consist in 2 sides: your own files and the ones shared with you.
```
Comment (fmartin-linagora) [inline documentation/EN/user/linshare-user.md]: consisted ? no ?

File: documentation/EN/user/linshare-user.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+</a>
+
+**My files** gather all the files you uploaded on _your space_. Each shared file (which is not past the expiration date) is labeled "_shared_".</br>
+To upload a new file, you have 2 choices:
+
+- Clic on the **+** button on the upper left side.
+- Drag and drop your file from your file browser to your own space (you can then see the center circle become blue).
+
+When a uploaded file is hoovered, 3 emojis appear on the right side. From left, those emojis mean:
```
Comment (fmartin-linagora) [inline documentation/EN/user/linshare-user.md]: emojis ? :) icons ?

File: documentation/EN/user/linshare-user.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+<a name="current-uploads">
+
+##### Current uploads
+
+</a>
+
+All current uploads are listed in this section. The current uploads gather those from [My space](#my-space) and [Shared space](#shared-space).
+To actually visualize the uploads on My space, or Shared space, you need to clic on the corresponding tab on the upper right side.
+
+Some actions are available in the upper right corner, near the _My sapce_ and _Shared space_ tabs. Those actions are the following:
```
Comment (fmartin-linagora) [inline documentation/EN/user/linshare-user.md]: sapce => space

**Comment by fmartin-linagora** [comment]
Fine for me, could you fix your little mistakes please ? :)
You may to rebase your work too.

File: documentation/EN/user/linshare-user.md (hunk 2 above)
Comment (RiceBowlJr) [inline documentation/EN/user/linshare-user.md]: It was late, I was a bit tired 👅

**Comment by RiceBowlJr** [comment]
Nevertheless, got a problem with my repo, won't pull the last commits on master, so I'll make another PR and then delete this one.

**Comment by RiceBowlJr** [comment]
Reviewed applied chef master ;)

**Pull request merged**